muxers/mplex/src/codec.rs

```diff
@@ -28,7 +28,7 @@ use unsigned_varint::{codec, encode};
 // Arbitrary maximum size for a packet.
```

Comment:
Oh, could you also please change this comment. It says "arbitrary maximum size", but it's in fact in the specs and not arbitrary.

Reply:
done